### DISCUSSION TITLE
Welcome to ThingsDB Discussions!

### DISCUSSION THREAD

**Discussion by joente** [thingsdb/ThingsDB#191] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joente** [comment]
Hi, I'm Jeroen. I've started the ThingsDB project and worked on the core for a few years. 

Besides the core development, I'm also active as maintainer for the ThingsDB clients and documentation.

**Comment by AnjaBruls** [comment]
Hi, I'm Anja. I'm active as maintainer of ThingsGUI which is a graphical web-interface to manage ThingsDB.

**Comment by ivomans** [comment]
Hi, I'm Ivo. I started using ThingsDB as the core platform for the project planning module in our ERP software. 

The many-to-many relations between typed things are very handy to connect all the stages of the projects to be planned.
The user sees a graphical representation of the planning with an Angular app. This app is a client to the planning-room (with [thingsdb.js](https://github.com/stefanak-michal/thingsdb.js)), thus any modification in the planning is live updated on all the screens of every user.
Modifications to the project elsewhere in our PHP/Mysql based ERP software is updated in the ThingsDB environment with [thingsdb-php](https://github.com/stefanak-michal/thingsdb-php).
We are still improving the UI of the app and expect to have the module in production soon.

The combination of handy data manipulation with communication in rooms is a very interesting concept for our business applications.

**Reply by joente** [reply]
Welcome Ivo!
